common/ownership_voucher/ownership_voucher.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
here and other file below, please add some whitespace to mirror the client.go file. header (Sorry i think i copied it to you without the whitespace)

//     https://www.apache.org/licenses/LICENSE-2.0